frontend/public/components/overview.jsx

```diff
@@ -264,13 +264,20 @@ class OverviewDetails extends React.Component {
     const {filteredItems, filterValue, groupedItems, groupOptions, selectedGroupLabel} = this.state;
 
     return <div className="co-m-pane">
-      <OverviewToolbar
-        filterValue={filterValue}
-        groupOptions={groupOptions}
-        handleFilterChange={this.handleFilterChange}
-        handleGroupChange={this.handleGroupChange}
-        selectedGroup={selectedGroupLabel}
-      />
+      <div className="co-m-nav-title project-overview__filter-groupby">
```

Comment:
There is no`project-overview` block, so `project-overview__filter-groupby` is out of compliance with BEM guidelines.  I recommend a modifier on `co-m-nav-title` (e.g., `co-m-nav-title--overview`).